package.json

```diff
@@ -10,9 +10,9 @@
     "start": "grunt start",
     "build": "grunt build",
     "eslint": "eslint .",
-    "test": "jest tests/ --runInBand --coverage",
-    "test-no-coverage": "jest tests/ --runInBand",
-    "test-ci": "npm run test && cat ./coverage/lcov.info | coveralls",
+    "test": "jest tests/ --runInBand --watch",
```

Comment:
Would be good to get to the bottom of why that is. Not using watch defeats one of the best features of Jest.